store/postgres/src/filter.rs

```diff
@@ -271,9 +271,72 @@ fn store_filter_by_mode<'a>(
         }
         // Is `attribute` equal to some `v` in `query_values`?
         StoreFilter::In(attribute, query_values) => {
-            query_values.into_iter().try_fold(query, |q, v| {
-                store_filter_by_mode(q, StoreFilter::Equal(attribute.clone(), v), FilterMode::Or)
-            })?
+            let op = " = ANY (";
+            match query_values[0].clone() {
```

Comment:
Is there anything that guarantees the list is not empty? Otherwise, this unconditional index access could panic.

Reply:
This will also be resolved by issue #272.